src/main.ts

```diff
@@ -62,6 +75,12 @@ const command: Command = {
 			default: false
 		});
 
+		options('i', {
```

Comment:
I think by default the `dojo/cli` test command should ship with default configuration for all Dojo 2 supported browsers and if a user needs to do something outside of that then they need to `eject`. It's about minimal configuration options providing intelligent defaults and offering the `eject` capability for advanced consumers.

In a slight aside, it feels like the `config` options is badly named and should perhaps be `target` or `environment` because a consumer has no idea that it actually relates to a configuration file internally.